application.properties

```diff
@@ -1,7 +1,7 @@
 #Grails Metadata file
-#Mon Jul 21 16:57:34 EST 2014
+#Tue Aug 01 15:35:23 AEST 2017
 app.buildNumber=35
-app.grails.version=2.3.11
+app.grails.version=2.5.6
 app.name=ala-downloads
 app.servlet.version=2.5
```

Comment:
Re: servlet version 3.0 - could you change the property `grails.servlet.version` on line 1 of `BuildConfig.groovy` as well?